azure-pipelines.yml

```diff
@@ -47,7 +47,7 @@ jobs:
           reportDirectory: '$(System.DefaultWorkingDirectory)/**/coverage'
   - job: Build
     pool:
-      vmImage: 'Ubuntu 18.04'
+      vmImage: 'Ubuntu 22.04'
```

Comment:
We might want to consider using our 1ES pools at some point for these, assuming they are in azure-sdk DevOps org.